ui/app/components/variable-paths.hbs

```diff
@@ -29,6 +34,10 @@
         data-test-file-row
         {{on "click" (fn this.handleFileClick file)}}
         class={{if (can "read variable" path=file.absoluteFilePath namespace=file.variable.namespace) "" "inaccessible"}}
+        {{keyboard-shortcut 
+          enumerated=true
+          action=(fn this.handleFileClick file)
```

Comment:
`blocking`: This should accept a namespace.

Reply:
Hi @ChaiWithJai — the `handleFileClick` action reads namespace from the variable (passed here as `file`) itself. As such , namespaces do not need to be passed in. This is the same logic we use a couple lines above this in the on "click".